docs/fluttertodostutorial.md

```diff
@@ -30,15 +30,20 @@ very_good create flutter_app flutter_todos --desc "An example todos app that sho
 
 Next we'll create the `todos_api`, `local_storage_todos_api`, and `todos_repository` packages using `very_good_cli`:
 
+```sh
```

Comment:
This isn’t necessary because specifying -o will generate the directories needed.